src/Persistence.Tests/_TestData/ValidYaml/Screen-with-BuiltInControl3.yaml

```diff
@@ -0,0 +1,4 @@
+ButtonCanvas: 
+Name: BuiltIn Control1
+Properties:
+  Text: 'Text`~!@#$%^&*()_-+=", ":'
```

Comment:
More suggested tests - one of the things that breaks using the planned `=` formula format is when values contain a colon or octothorp _proceeded_ by a space - `"Hello : World"` and `"Hello # World"`